working/augmentation-libraries/feature-specification.md

```diff
@@ -142,13 +142,32 @@ source order, and then recursing on `part`-directives in source order.
 
 ## Augmentation declarations
 
-Augmentation declarations are declarations marked with the new built-in
-identifier `augment`, which makes the declaration augment an existing
-declaration, which is itself a normal declaration with zero or more prior
-augmentations applied. An augmentation declaration does not  introduce a new
-name into the surrounding scope, it attaches itself to the existing name.
+Every declaration in Dart without this feature is _not_ an augmentation
```

Comment:
This reads a bit weird to me. Maybe `Every declaration without the `augment` keyword is referred to as an _introductory_ declaration.`?.

Reply:
True, that's weird! I rewrote the paragraph entirely.